README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
 
 ISO/IEC 8652 is officially maintained by ISO/IEC JTC 1/SC 22/WG 9. Comments from the general public are handled by WG 9's Ada Rapporteur Group (ARG). The ARG is responsible for addressing these comments, which often concern possible enhancements or errors in the Ada standard. The ARG considers these enhancements or errors for incorporation into future versions of the Ada standard.
```

Comment:
I never noticed this sentence before, but it implies we are incorporating "errors" into future versions of the Ada standard.  I think it would be better to say something like:

> The ARG considers proposals that address these identified shortcomings for incorporation into future versions of the Ada standard.

Reply:
Good point, this sentence is awkward. As for your suggested change, I wonder if "these" is a bit too harsh in some sense, maybe "any" is better?

> The ARG considers proposals that address _any_ identified shortcomings for incorporation into future versions of the Ada standard.